app/models/case_log.rb

```diff
@@ -480,6 +481,19 @@ def get_lettype
     end
   end
 
+  def get_housingneeds
```

Comment:
Any way of adding a bit more context/meaning to this method?

Reply:
@baarkerlounger I've updated this, let me know if this is better?

Reply:
Nice :+1: